app/main.py

```diff
@@ -1,8 +1,44 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int, clean_mark: int, brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self, distance_from_city_center: int,
+            clean_power: int, average_rating: int,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_the_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+
+        return round(income, 1)
+
+    def wash_single_car(self, car: Car) -> None:
+        car.clean_mark = self.clean_power
+
+    def calculate_washing_price(self, car: Car) -> float:
+        clean_difference = self.clean_power - car.clean_mark
+        factor = self.average_rating / self.distance_from_the_city_center
+        single_car_wash_price = (
+            car.comfort_class * clean_difference * factor
+        )
+        return round(single_car_wash_price, 1)
```

Comment:
Code Style Issue: In the `calculate_washing_price` method, you have a long mathematical expression. This can be split into additional variables or broken down after binary operations for better readability. Please refer to the checklist for examples.